fenix/app/src/main/java/org/mozilla/fenix/library/history/state/HistoryNavigationMiddleware.kt

```diff
@@ -72,6 +75,15 @@ class HistoryNavigationMiddleware(
                         }
                     }
                 }
+                is HistoryFragmentAction.SearchClicked -> {
+                    val directions = if (settings.showUnifiedSearchFeature) {
```

Comment:
Let's remove this check for `showUnifiedSearchFeature`. I am trying to remove all of this in https://github.com/mozilla-mobile/firefox-android/pull/3352.

Reply:
Will do, thanks for the catch